### PR TITLE
Enable production loggers and logger for GCP

### PR DESCRIPTION
**Pull request by AdamMagaluk**
NewLoggerForGCP creates a new logger formatted for Cloud Logging with fields and level correctly set.

Production log level should be Info and Development: false

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -46,6 +47,30 @@ var defaultProductionConfig = zap.Config{
 	ErrorOutputPaths:  []string{"stderr"},
 }
 
+// from https://github.com/uber-go/zap/blob/2314926ec34c23ee21f3dd4399438469668f8097/config.go#L98
+// but disable stacktraces.
+var defaultProductionGCPCloudConfig = zap.Config{
```
Comment (edaniels) [inline logger.go:52]: maybe call this a cloud config, not gcp?

**Review by edaniels — APPROVED** [review]

File: logger.go (hunk 1 above)
Comment (AdamMagaluk) [reply logger.go:52]: Its specific to how GCP Cloud Monitoring processes the fields for msg, severity etc... So i'm not sure generic Cloud is valid.

Comment (edaniels) [reply logger.go:52]: okay cool

Comment (edaniels) [reply logger.go:52]: sorry future eric for adding google stuff

**Pull request merged**